cypress/integration/bundled-osd/check_timeline.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/* es-lint-disable for missing definitions */
+/* eslint-disable */
+import { MiscUtils, LoginPage } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+const loginPage = new LoginPage(cy);
+  
+describe('verify timeline visualization work properly for bwc', () => {
+    beforeEach(() => {
+        miscUtils.visitPage('app/visualize#');
+        loginPage.enterUserName('admin');
+        loginPage.enterPassword('admin');
+        loginPage.submit();
+    });
+
+    afterEach(() => {
+        cy.clearCookies();
+    });
+
+    it('tenant-switch-modal page should show and be clicked', () => {
+        cy.get('[data-test-subj="tenant-switch-modal"]');
+        cy.get('[data-test-subj="confirm"]').click();
+    });
+    
+    it('timeline visualizations should be saved and named correctly', () => {
+        cy.get('[data-test-subj="visualizationLandingPage"]')
+          .find('[class="euiFormControlLayout__childrenWrapper"]')
+          .type('timeline');       
+        cy.get('[data-test-subj="visListingTitleLink-test-timeline"]').should('have.text', 'test-timeline').click();
+        cy.get('[class="view-line"]').contains('.es(*)');
+    });
+
+    describe('timeline visualizations should work properly', () => {
+        beforeEach(() => {
+           cy.get('[data-test-subj="visualizationLandingPage"]')
+             .find('[data-test-subj="newItemButton"]')
+             .click();     
+           cy.get('[data-test-subj="visType-timelion"]').click();
+         });
+
+         it('.es(*, kibana1=true should report search error', () => {
```

Comment:
nit: same as the above. For example `.opensearch(*, kibana=true)` and `.opensearch(*, opensearchDashboards=true)`

Reply:
thanks for adding.